detox/test/e2e/05.waitfor.test.js

```diff
@@ -11,18 +13,17 @@ describe('WaitFor', () => {
     await expect(element(by.id('createdAndVisibleText'))).toExist();
   });
 
-  it('should wait until an element is removed', async() => {
+  it('should wait until an element is removed', async () => {
     await expect(element(by.id('deletedFromHierarchyText'))).toBeVisible();
     await element(by.id('GoButton')).tap();
     await waitFor(element(by.id('deletedFromHierarchyText'))).toBeNotVisible().withTimeout(20000);
     await expect(element(by.id('deletedFromHierarchyText'))).toBeNotVisible();
   });
 
-  it('should find element by scrolling until it is visible', async() => {
+  custom.it.withFailureIf.android.rn58OrNewer('should find element by scrolling until it is visible', async () => {
```

Comment:
What is the point of this if we know it will fail anyway? We will be running the RN59 suite from now on, on both iOS and Android. Why not just close it off for Android until we have a proper fix?

Reply:
This is so we'd know when (if) they decide to get it fixed.